cfn/configs/prod/us-east-1/config.yml

```diff
@@ -0,0 +1,530 @@
+  #
+  # o_container: list of dictionaries
+  #   Using the same "dictionary" object for container o_container is used to describe the 2..N docker containers
+  #   to be associated with the ECS task definition.
+
+  container:
+    name: !_join ["-", *SERVICE_NAME, *ENVIRONMENT]
+    essential: true
+    memory_reservation: 512
+    logs: awslogs
+
+    # name: string
+    #   used to name the ECS Container within the task definition.
+    #   Also used to name the Docker image if neither container.image or service.image are defined
+
+    # essential: bool (default True)
+    #   used by ECS to determine what action to take if the container terminates.
+
+    # memory_reservation: int (default 512, expressed in MB)
+    #   Used during task "placement" to determine how much memory the task (service-instance) will be consumed.
+    #   Is not enforced in anyway, purely for planning purposes.
+
+    # logs: string (default awslogs)
+    #   This is used to define how logs are collected and processed by ECS. This is both a "generator" for
+    #   "awslogs" and "fluentd", otherwise allows the user to pass parameters directly to the ECS::Task
+    #
+    #   When "awslogs":
+    #     All logs are forwarded to CloudWatch logs
+    #     awslogs-group is StackName
+    #     awslogs-region is Region
+    #     awslogs-stream-prefix is environment
+    #
+    #   When "fluentd":
+    #     Logs are forwarded to FluentD process running on EC2 hosts as part of cluster 172.17.0.1:24224
+    #     fluent-tag is service.name-service.environment with the docker container ID appended
+    #     Additional config object container.log_options is a dictionary allowing other params to be provided as options
+    #
+    #   Otherwise:
+    #     LogDriver = container.logs
+    #     Options is dictionary based on container.log_options
+
+    # By default, the image uses your container name and a parameter stored at /deployments/container_name/latest
+    # If you use the build process included in this repo, it will automatically store the latest version for you in that parameter
+    #
+    # image = container_name
+
+    # If you are using a custom image version tag, put that here
+    # otherwise the default is via SSM is /deployments/app/latest/
+    # If the string begins with a "/" it is expected to be a SSM param name, otherwise its a plain string
+    #
+    # image_version: /deployments/app/VERSION_NAME
+    # image_version: 0123897107860fed (Git Commit Hash)
+
+    # If you need a custom image not managed in SSM
+    # If you do this, you will need to make sure to include code to authenticate with the provider
+    #
+    # image = provider/project/repo:tag
+    #         In order for "image" to be reconized as raw_image_path, it must contain a "/" or a ":"
+    #
+    # raw_image_path: customer_path_to_image (Deprecated)
+
+
+    environment:
+      DEBUG: yes
+      ## TODO: @Katlyn, set up the env vars here
```

Comment:
TODO @Kat-Alo 

Also check out if you need the secrets manager block just after this.